<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>S3FileBrowser</class>
 <widget class="QDialog" name="S3FileBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>S3 File Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="bucketNameEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>S3 Bucket Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="objectList"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>S3FileBrowser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>S3FileBrowser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>objectList</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>S3FileBrowser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>S3FileBrowser</receiver>
   <slot>refreshButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>objectList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>S3FileBrowser</receiver>
   <slot>objectList_itemSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
